media/articles/appliance/modules.md

```diff
@@ -4,35 +4,5 @@ For security reasons, rules and custom database connections for Auth0 appliances
 
 The current sandbox supports:
 
-* [async](https://github.com/caolan/async) _(~0.9.0)_
```

Comment:
- Do we no longer support these packages? Or does this no longer apply? This appears to be a list of **libraries**, not magic globals.
- The version of `auth0` appears to be quite outdated, is `2.0.0-alpha.5` really what we have for Node 8/12 webtasks?

Reply:
These look like the names of the magic globals that we expose. For some, like `sqlserver` below, it's backed by the _library_ called `tedious`.

And yes, unfortunately, the `auth0` magic global is pinned to `2.0.0-alpha.5` to maintain backwards compatibility with rules. We _also_ expose a capital-A `Auth0` [here](https://github.com/auth0/auth0-authz-rules-api/blob/master/index.js#L242-L248) that will be the most recent version. That version should be at `2.23.0` now in both Node 8 and Node 12.

Reply:
Yes. The title of this doc is `Node.js Modules Available in Rules and Custom Database Connections` and I just noticed that it's for appliances only (PSaaS). This page lives at https://auth0.com/docs/appliance/modules.

Maybe we shouldn't change this yet?

Reply:
I agree
I will not change this file.